dizoo/bsuite/config/serial/memory_len/memory_len_10_r2d2_config.py

```diff
@@ -0,0 +1,79 @@
+from easydict import EasyDict
+from ding.entry import serial_pipeline
+
+collector_env_num = 8
+evaluator_env_num = 1
+memory_len_r2d2_config = dict(
+    exp_name='memory_len_10_r2d2',
+    env=dict(
+        collector_env_num=collector_env_num,
+        evaluator_env_num=evaluator_env_num,
+        n_evaluator_episode=100,
+        env_id='memory_len/10',  # 12 memory steps, 13 obs per episode
+        stop_value=1.,
+    ),
+    policy=dict(
+        cuda=True,
+        on_policy=False,
```

Comment:
Please remove this in the config as this key should be in its default setting in the corresponding policy

Reply:
Got it, thank you